doc/mgmt/Docker to Host communication.md

```diff
@@ -2,7 +2,7 @@
 Docker to Host Communications
 
 # High Level Design Document
-#### Rev 0.7
+#### Rev 0.8
```

Comment:
Original document mentioned "If multiple D-Bus clients simultaneously issue requests, then each request is queued and
processed in the order received". Is it possible to the requests could be conditional processed in parallel?

Reply:
this is dbus limitation, it can't process multiple requests for the same service in parallel.